openproblems/tasks/_batch_integration/batch_integration_graph/methods/__init__.py

```diff
@@ -0,0 +1,69 @@
+from .bbknn import bbknn_full_scaled
+from .bbknn import bbknn_full_unscaled
+from .bbknn import bbknn_hvg_scaled
+from .bbknn import bbknn_hvg_unscaled
+from .combat import combat_full_scaled
+from .combat import combat_full_unscaled
+from .combat import combat_hvg_scaled
+from .combat import combat_hvg_unscaled
+
+# from .desc import desc_full_scaled
+# from .desc import desc_full_unscaled
+# from .desc import desc_hvg_scaled
+# from .desc import desc_hvg_unscaled
+from .fastmnn_embed import fastmnn_embed_full_scaled
+from .fastmnn_embed import fastmnn_embed_full_unscaled
+from .fastmnn_embed import fastmnn_embed_hvg_scaled
+from .fastmnn_embed import fastmnn_embed_hvg_unscaled
+
+# from .fastmnn_feature import fastmnn_feature_full_scaled
+# from .fastmnn_feature import fastmnn_feature_full_unscaled
+# from .fastmnn_feature import fastmnn_feature_hvg_scaled
+# from .fastmnn_feature import fastmnn_feature_hvg_unscaled
+from .harmony import harmony_full_scaled
+from .harmony import harmony_full_unscaled
+from .harmony import harmony_hvg_scaled
+from .harmony import harmony_hvg_unscaled
+from .liger import liger_full_unscaled
+from .liger import liger_hvg_unscaled
+from .mnn import mnn_full_scaled
+from .mnn import mnn_full_unscaled
+from .mnn import mnn_hvg_scaled
+from .mnn import mnn_hvg_unscaled
+
+# from .saucie_embed import saucie_embed_full_scaled
+# from .saucie_embed import saucie_embed_full_unscaled
+# from .saucie_embed import saucie_embed_hvg_scaled
+# from .saucie_embed import saucie_embed_hvg_unscaled
+# from .saucie_full import saucie_feature_full_scaled
+# from .saucie_full import saucie_feature_full_unscaled
+# from .saucie_full import saucie_feature_hvg_scaled
+# from .saucie_full import saucie_feature_hvg_unscaled
```

Comment:
why no saucie?

Reply:
What's wrong with SAUCIE and FastMNN?

Reply:
So this PR is without saucie and scgen although you added the dockerfiles? Ideally remove those docker images then and put them in the next PR (as they're not tested here).